Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: site
 site:
-	soupault ${SOUPAULT_OPTS}
+	./soupault ${SOUPAULT_OPTS}
```

Comment:
This will break the CI, since the CI workflow installs soupault system-wide, as it should (well, because it can and doesn't have issues like WSL). Please revert this change.